ansible/roles/flannel/tasks/firewalld.yml

```diff
@@ -0,0 +1,5 @@
+---
+# https://bugzilla.redhat.com/show_bug.cgi?id=1033606 and I think others say firewalld+docker == bad
+# It is also disabled by the node role.
+- name: disable firewalld
+  service: name=firewalld enabled=no state=stopped
```

Comment:
Is there an actual incompatibility here? Docker has been fixed. If there is a flannl incompatibility here fix it, don't paper over it. Let kube turn off firewalld (as it does today) until https://github.com/kubernetes/kubernetes/pull/12396 lands.